dist/interface.js

```diff
@@ -292,18 +307,40 @@ var app = new Vue({
     prepareToSaveData: function prepareToSaveData(stopComplete, imageSaved) {
       var _this4 = this;
 
+      this.hasError = false;
+      this.hasErrorOnSave = false;
+
       if (!stopComplete && !this.maps.length) {
-        this.hasErrorOnSave = true;
+        this.hasErrorOnSave = {
+          message: 'You need to add at least one map to save and close.'
+        };
         return;
       }
 
       if (stopComplete && !imageSaved && (!this.maps.length || !this.markers.length)) {
-        this.hasError = true;
+        this.hasError = {
+          message: 'You need to add at least one map before continuing.'
+        };
         return;
-      }
+      } // Check if maps have images
```

Comment:
why is this comment here 🤔

Reply:
@squallstar that's the transpiled JS. Not sure why but the system put comments inline.